multy_core/src/bitcoin/bitcoin_account.cpp

```diff
@@ -117,6 +117,65 @@ class BitcoinP2PKHAccount : public BitcoinAccount
     }
 };
 
+class BitcoinP2SHWPKHAccount : public BitcoinAccount
+{
+public:
+    using BitcoinAccount::BitcoinAccount;
+
+    std::string get_address() const override
+    {
+
```

Comment:
This looks like a step-by-step guide from some documentation, could you please paste a link function header comment to that doc?